src/interface/viewport.c

```diff
@@ -675,6 +676,10 @@ void viewport_render(rct_drawpixelinfo *dpi, rct_viewport *viewport, int left, i
  */
 void viewport_paint(rct_viewport* viewport, rct_drawpixelinfo* dpi, int left, int top, int right, int bottom){
 	gCurrentViewportFlags = viewport->flags;
+	//This should still be updated until the rest of supports, etc, is reverse-engineered. Otherwise
+	// Wooden Coaster supports (and a few other coaster supports) will still appear even if "invisible supports"
```

Comment:
All coasters apart from jr rc. I wouldn't bother drawing attention to just wooden coasters just say unimplemented coasters.